docs/src/modules/ROOT/pages/quickstart/spring-boot/spring-boot-quickstart.adoc

```diff
@@ -16,7 +16,7 @@ with https://timefold.ai[Timefold]'s constraint solving Artificial Intelligence
 
 You will build a REST application that optimizes a school timetable for students and teachers:
 
-image::quickstart/school-timetabling/schoolTimetablingScreenshot.png[]
+image::quickstart/school-timetabling/schoolTimetablingScreenshot_legacy.png[]
```

Comment:
Why do we keep and reference an *old* version of the app?

Reply:
The other quickstarts, including the SpringBoot one, have not been converted to the new format.